pkg/api/v1/types.go

```diff
@@ -20,6 +20,7 @@ import (
 	"k8s.io/client-go/pkg/types"
 	"kubevirt.io/kubevirt/pkg/api"
 	"kubevirt.io/kubevirt/pkg/precond"
+	"kubevirt.io/kubevirt/pkg/util"
```

Comment:
It is called "mapper" now, since after the refactoring only mapper code is left in there.